hack/list-oci-artifacts.sh

```diff
@@ -1,42 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This script is used in `kommander` repo. The bundle creation make target will require an update.